.salsaflow/config.yml

```diff
@@ -0,0 +1,40 @@
+# This field keeps the timestamp of when 'repo bootstrap' was executed.
```

Comment:
I left a comment about this one or two days ago, but I have actually no idea where it went :suspect: 

So for the record: Please leave `.salsaflow` out of the upstream repository (i.e. this one), it doesn't make sense to anyone outside Salsita. Should suffice if you keep it in your fork.